tests/rules/test_new_lines.py

```diff
@@ -58,3 +61,27 @@ def test_dos_type(self):
         self.check('\r\n', conf)
         self.check('---\ntext\n', conf, problem=(1, 4))
         self.check('---\r\ntext\r\n', conf)
+
+    def test_platform_type(self):
+        conf = ('new-line-at-end-of-file: disable\n'
+                'new-lines: {type: platform}\n')
+
+        self.check('', conf)
+
+        # mock the Linux new-line-character
+        with mock.patch("yamllint.rules.new_lines.linesep", '\n'):
```

Comment:
Let's use single quotes for consistency :+1: 
(same below)

Reply:
yes, also saw that while testing the import XD

fixed